olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp

```diff
@@ -237,25 +237,30 @@ bool PartitionsCacheRepository::Get(geo::TileKey tile_key, int32_t depth,
   return false;
 }
 
-void PartitionsCacheRepository::Clear() {
+bool PartitionsCacheRepository::Clear() {
   auto key = catalog_ + "::" + layer_id_ + "::";
   OLP_SDK_LOG_INFO_F(kLogTag, "Clear -> '%s'", key.c_str());
-  cache_->RemoveKeysWithPrefix(key);
+  return cache_->RemoveKeysWithPrefix(key);
 }
 
-void PartitionsCacheRepository::ClearPartitions(
+bool PartitionsCacheRepository::ClearPartitions(
     const std::vector<std::string>& partition_ids,
     const boost::optional<int64_t>& version) {
   OLP_SDK_LOG_INFO_F(kLogTag, "ClearPartitions -> '%s'", catalog_.c_str());
   auto cached_partitions = Get(partition_ids, version);
+  bool suceeded = true;
 
   // Partitions not processed here are not cached to begin with.
   for (const auto& partition : cached_partitions.GetPartitions()) {
-    cache_->RemoveKeysWithPrefix(catalog_ + "::" + layer_id_ +
-                                 "::" + partition.GetDataHandle());
-    cache_->RemoveKeysWithPrefix(catalog_ + "::" + layer_id_ +
-                                 "::" + partition.GetPartition());
+    suceeded = suceeded &&
+               cache_->RemoveKeysWithPrefix(catalog_ + "::" + layer_id_ +
+                                            "::" + partition.GetDataHandle());
+    suceeded = suceeded &&
+               cache_->RemoveKeysWithPrefix(catalog_ + "::" + layer_id_ +
+                                            "::" + partition.GetPartition());
```

Comment:
Maybe we should reorder the check, in case suceeded is false, second term is not evaluated. Also `succeeded`

Reply:
Done. Replaced `succeeded` with `passed`, otherwise code formatting is making it even less readable than it is